planning/freespace_planner/src/freespace_planner/freespace_planner_node.cpp

```diff
@@ -429,13 +438,20 @@ void FreespacePlannerNode::onTimer()
 
   initializePlanningAlgorithm();
   if (isPlanRequired()) {
-    reset();
-
     // Stop before planning new trajectory
-    const auto stop_trajectory = createStopTrajectory(current_pose_);
-    trajectory_pub_->publish(stop_trajectory);
-    debug_pose_array_pub_->publish(trajectory2PoseArray(stop_trajectory));
-    debug_partial_pose_array_pub_->publish(trajectory2PoseArray(stop_trajectory));
+    if (partial_trajectory_.points.size() > 0) {
+      const auto stop_trajectory = createStopTrajectory(partial_trajectory_);
+      trajectory_pub_->publish(stop_trajectory);
+      debug_pose_array_pub_->publish(trajectory2PoseArray(stop_trajectory));
+      debug_partial_pose_array_pub_->publish(trajectory2PoseArray(stop_trajectory));
+    } else {
+      const auto stop_trajectory = createStopTrajectory(current_pose_);
+      trajectory_pub_->publish(stop_trajectory);
+      debug_pose_array_pub_->publish(trajectory2PoseArray(stop_trajectory));
+      debug_partial_pose_array_pub_->publish(trajectory2PoseArray(stop_trajectory));
+    }
```

Comment:
This seems a little redundant. How about write like
```suggestion
    const auto stop_trajectory = partial_trajectory_.points.empty()
                                   ? createStopTrajectory(current_pose_)
                                   : createStopTrajectory(partial_trajectory_);
    trajectory_pub_->publish(stop_trajectory);
    debug_pose_array_pub_->publish(trajectory2PoseArray(stop_trajectory));
    debug_partial_pose_array_pub_->publish(trajectory2PoseArray(stop_trajectory));
```

Reply:
good suggestion~

Reply:
hmm, this commit was dropped?

Reply:
> hmm, this commit was dropped?

Sorry, I revert some changes due to DCO issue and I didn't notice that. Now I think recommit this changes.

Reply:
Thaks for recommiting!